### PR TITLE
Feature: Add New Outputs (EmulationStation + RetroDECK)

### PR DESCRIPTION
**Pull request by maxexcloo**
Let me know if there's anything I need to add or change!

### CODE REVIEW

**Review by emmercm — CHANGES_REQUESTED** [review]

File: src/types/gameConsole.ts [hunk 1]
```diff
@@ -66,123 +75,154 @@ export default class GameConsole {
       mister: 'AcornAtom',
     }),
     // Amstrad
-    new GameConsole(/CPC/i, [], {
+    new GameConsole(/CPC/i, ['.cdt', '.cpr', '.dsk', '.kcr', '.m3u', '.sna', '.tap', '.tar', '.voc'], {
```
[the remark this answers is not in the document]
Comment (maxexcloo) [reply src/types/gameConsole.ts:78]: Will revert any that aren’t unique - but what if the extensions are the same between consoles? MSX and MSX2 for example?

File: src/types/outputFactory.ts [hunk 2]
```diff
@@ -270,6 +270,11 @@ export default class OutputFactory {
       output = output.replace('{adam}', adam);
     }
 
+    const emulationstation = gameConsole.getEmulationStation();
+    if (emulationstation) {
+      output = output.replace('{emulationstation}', emulationstation);
```
[the remark this answers is not in the document]
Comment (maxexcloo) [reply src/types/outputFactory.ts:275]: Will do!

File: src/modules/argumentsParser.ts [hunk 3]
```diff
@@ -753,13 +753,15 @@ Advanced usage:
 
     {adam}      The ROM's emulator-specific /ROMS/* directory for the 'Adam' image (e.g. "GB")
     {batocera}  The ROM's emulator-specific /roms/* directory for Batocera (e.g. "gb")
+    {emulationstation} The ROM's emulator-specific /roms/* directory for the 'EmulationStation' image (e.g. "gb")
```
[the remark this answers is not in the document]
Comment (maxexcloo) [reply src/modules/argumentsParser.ts:756]: How about ES-DE? I’ve seen that abbreviation before - I totally agree on the width btw

**Comment by maxexcloo** [comment]
@emmercm have updated and incorporated your feedback - let me know if there's anything else I'm missing :)

File: src/modules/argumentsParser.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (emmercm) [reply src/modules/argumentsParser.ts]: I'm great with `{es}`!

**Comment by emmercm** [comment]
@maxexcloo thank you for addressing everything! I made a few small changes so that I can include this in tonight's release: https://github.com/emmercm/igir/pull/943/files/e4b29c8e8329154a264fa15dc6aa5cb7d9480c3f..7d3cf4c9725c223a2e5fdd9354ce412bde6345e4

**Comment by maxexcloo** [comment]
Apologies for the errors there, appreciate the extra pair of eyes and the help merging this 💯

**Pull request merged**